CMakeLists.txt

```
@@ -5,8 +5,16 @@ cmake_minimum_required(VERSION 3.16)
 
 project(lskv LANGUAGES C CXX)
 
-if(NOT TARGET ccf)
-  find_package(ccf REQUIRED)
+option(CCF_UNSAFE "build with unsafe options" OFF)
+if(CCF_UNSAFE)
+  message(WARNING "Building with unsafe options")
+  if(NOT TARGET ccf_unsafe)
+    find_package(ccf_unsafe REQUIRED)
+  endif()
+else()
+  if(NOT TARGET ccf)
+    find_package(ccf REQUIRED)
+  endif()
```

```suggestion
find_package(ccf_unsafe REQUIRED)
else()
find_package(ccf REQUIRED)
```

Should also work I believe. The extras `if(NOT TARGET...)` are only there as a leftover from the logging app in the CCF repo that should be buildable both standalone and in the CCF build.